example/src/samples/sample-data.ts

```diff
@@ -712,3 +713,70 @@ export const exampleDownloadFile: ChatItem = {
         filePaths: ['Refactor_analysis_[id] .pdf'],
     },
 };
+
+export const exampleInformationCard: ChatItem = {
+    messageId: generateUID(),
+    type: ChatItemType.ANSWER,
+    informationCard: {
+        title: 'Information card',
+        description: 'With a description below the title.',
+        icon: MynahIcons.BUG,
+        content: {
+            body: sampleList2 as string
+        },
+    },
+};
+
+export const exampleInformationCardWarning: ChatItem = {
```

Comment:
nit: Maybe just a function to send changing parameters instead of creating multiple ones.